packages/flutter_app/lib/features/pub_dev_packages/view/pub_dev_packages_page.dart

```diff
@@ -0,0 +1,207 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:gap/gap.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import '../../../gen/strings.g.dart';
+import '../query/query.dart';
+
+/// Pub.dev packages page.
+class PubDevPackagesPage extends ConsumerWidget {
+  const PubDevPackagesPage({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final t = Translations.of(context);
+    final asyncValue = ref.watch(pubDevPackagesPageStateProvider);
+
+    return GestureDetector(
+      behavior: HitTestBehavior.opaque,
+      onTap: FocusScope.of(context).unfocus,
```

Comment:
![badge](https://img.shields.io/badge/review-imo-important)

The `UnfocusOnTap` widget could be used👍

Reply:
I’ve updated the code to use the UnfocusOnTap widget.